packages/html-elements/src/elements/Anchor.tsx

```diff
@@ -19,5 +20,6 @@ export const A = forwardRef(({ href, target, ...props }: LinkProps, ref) => {
       },
     },
   });
-  return <Text accessibilityRole="link" {...props} {...nativeProps} ref={ref} />;
```

Comment:
The one thing I'm not sure about is the `ref` here. I fixed it in my own app with `ref={ref as any}`. I know that isn't ideal, but I'm not sure how else to allow the ref to pass to either `View` or `Text`.

Reply:
Try copying `<LI />`